### DISCUSSION TITLE
Is customAuthDomain setter available?

### DISCUSSION THREAD

**Discussion by devberkay** [firebase/flutterfire#13069] [Q&A] [answered]
I can't find how to use this pr :

https://github.com/firebase/flutterfire/pull/12218

I am using firebase_auth: ^4.19.6 on my flutter app.

here is changelog of ^4.18.0:

FEAT(firebase_auth): add custom auth domain setter to Firebase Auth ([#12218](https://github.com/firebase/flutterfire/issues/12218)). ([e1297800](https://github.com/firebase/flutterfire/commit/e12978009e0fd785f267db560972ab0bbe021fcb))

**Comment by Ehesp** [comment, marked as answer]
You should be able to call the setter on the auth instance:

```
final auth = FirebaseAuth.instance;
auth.customAuthDomain = '....';
```
